fe1-web/components/QRCode.tsx

```diff
@@ -1,23 +1,46 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import TextBlock from 'components/TextBlock';
 import { KeyPairStore } from 'store/stores';
+import QRCodeDisplay from 'qrcode.react';
+import { View } from 'react-native';
+import styleContainer from 'styles/stylesheets/container';
+import TextBlock from './TextBlock';
+
+/**
+ * Creates and displays a QR code with the public key of the current user
+ */
 
 const QRCode = (props: IPropTypes) => {
   const identity = KeyPairStore.getPublicKey().toString();
   const { visibility } = props;
+  const { size } = props;
+  const { text } = props;
 
-  // For now, we simply display a text block with the public key
+  // Displays a QR code with the Public key of the current user
   return (visibility)
-    ? <TextBlock bold text={`ID (pk) :   ${identity}`} />
+    ? (
+      <>
+        <TextBlock text={String(text)} />
+        <View style={[styleContainer.topCenter, { padding: 10 }]}>
+          <QRCodeDisplay value={identity} size={Number(size)} />
+        </View>
```

Comment:
I really tried to use something that existed already but wasn't able to get it looking the way I wanted.
If we use the `topCenter` it looks like this:
![image](https://user-images.githubusercontent.com/28142119/110386286-6387ec00-8060-11eb-9fdf-1ef7a953ea7b.png)

If we use `anchoredCenter` then it looks like this:
![image](https://user-images.githubusercontent.com/28142119/110386449-95994e00-8060-11eb-82a2-3c42456c4e82.png)

Which I found looked better since it was near the text and not centered in the middle of the page.

But the same outcome could be made by overwritting it in the component like this:
`<View style={[styleContainer.anchoredCenter, { padding: 10, justifyContent: 'flex-start' }]}>
          <QRCodeDisplay value={identity} size={Number(size)} />
        </View>`
Would that be better?